.github/workflows/pr-test.yml

```diff
@@ -377,6 +377,33 @@ jobs:
           cd python
           pytest -s -v --log-cli-level=INFO sglang/multimodal_gen/test/server/test_server_performance.py
 
+
+  multimodal-gen-test-2-gpu:
+    needs: [check-changes, sgl-kernel-build-wheels]
```

Comment:
Can we put this after multimodal-gen-test to follow the current convention to run 2-gpu-test only after 1-gpu-test finishes?